apartment.rb

```diff
@@ -1,2 +1,29 @@
+require_relative 'square_footage'
+
 class Apartment
+  include SquareFootage
+  attr_reader :rooms, :monthly_rent, :number
+
+
+  def initialize(args = {})
+    @rooms = args.fetch(:rooms, [])
+    @number = args.fetch(:number, nil)
+    @monthly_rent = args.fetch(:monthly_rent, 1000)
+  end
+
+  def collection_of_rooms
+    self.rooms
+  end
+
+  def price_per_sqft
+    self.monthly_rent.to_f / total_sqft
```

Comment:
Again, your attr_reader methods create getter methods for you, so it is not necessary to call self.monthly_rent. You can simply call monthly_rent.to_f / total_sqft. @perohrstrom